pages/style-guide.md

```diff
@@ -2,11 +2,11 @@
 	"title": "jQuery's Style Guides"
 }</script>
 
-jQuery projects strive to use a common set of styles for code and documentation. This facilitates a team of developers working on the same code base. If each developer were to use their own style, there would be jarring changes in the code as someone read through it. This slows comprehension and can also cause spurious differences in changesets as developers alter style in frivolous ways on the code they touch.
+Style guides aim to make a code base appear as if it were written by one precise punctilious programmer, rather than a cacophony of competing coders. As with many aspects of coding and writing, there can be differences of opinion about the best style. Contentious issues like "spaces versus tabs" or "semicolons or not" distract from the goal of writing code that works.
```

Comment:
What's "punctilious"? If I need to look at up first, it probably shouldn't be used here...